consumer/app/controllers/cats_controller.rb

```diff
@@ -2,4 +2,15 @@ class CatsController < ApplicationController
   def index
     @cats = ANIMALS.cats.list
   end
+
+  def create
+    ANIMALS.cats.create(cat_params)
```

Comment:
The consumer changes are minimal and straight-forward. All complexity is hidden behind the client library.